subprojects/security/src/main/java/org/gradle/security/internal/PGPUtils.java

```diff
@@ -17,13 +17,16 @@
 package org.gradle.security.internal;
 
 import org.bouncycastle.openpgp.PGPPublicKey;
+import org.bouncycastle.openpgp.PGPPublicKeyRing;
 
+import java.lang.reflect.Field;
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.List;
 
 public final class PGPUtils {
 
+    private static final Field KEYS_FIELD = getKeysField();
```

Comment:
KEY_FIELDS would sound much better to me

Reply:
But it's only one filed called "keys"